CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# How to contribute to the ODA Canvas Accelerator
+
+## Have you read our code of conduct?
+
+We want the ODA Canvas Accelerator to be project to be a welcoming community project. We have a [code of conduct](code-of-conduct.md) that we expect all participants to follow in all interactions with, or about, the project. We will take corrective action where the code of conduct is not followed.
```

Comment:
code-of-conduct.md is in subfolder charts.
Is this on purpose?
The link to the file is without "charts/" subfolder